src/v2/Apps/Order/Components/MinPriceWarning.tsx

```diff
@@ -22,19 +22,15 @@ export const MinPriceWarning: React.FC<MinPriceWarningProps> = ({
       flow: AnalyticsSchema.Flow.MakeOffer,
       order_id: orderID,
     })
-  }, [])
+  }, [orderID, tracking])
 
   return (
     <Clickable cursor="pointer" onClick={onClick} mt={2}>
       <Message variant="warning" p={2}>
         <Text>
-          Galleries usually accept offers within
           {isPriceRange
```

Comment:
Strange that we don't test both of these use cases.

Reply:
Actually, they were in a different component, fixed and improved in this [commit](https://github.com/artsy/force/pull/9714/commits/f5053a9051b2afb65493b665acdef18564cde3ca)